Python/Product/PyDebugAttach/PyDebugAttach.vcxproj

```diff
@@ -75,6 +75,8 @@
       <SubSystem>Console</SubSystem>
       <OutputFile>$(TargetPath)</OutputFile>
       <AdditionalDependencies>ws2_32.lib;psapi.lib;kernel32.lib;user32.lib;gdi32.lib;winspool.lib;comdlg32.lib;advapi32.lib;shell32.lib;ole32.lib;oleaut32.lib;uuid.lib;odbc32.lib;odbccp32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <GenerateDebugInformation Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">DebugFull</GenerateDebugInformation>
+      <FullProgramDatabaseFile Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">true</FullProgramDatabaseFile>
```

Comment:
Any reason to have the conditions in here? We always want these settings, right?

Reply:
No particular reason, it's just the way VS made it when using project properties.

It actually occurs to me that this project doesn't need it, while the one that does (`DebuggerHelper`), I forgot to stage.